pyannote/audio/core/inference.py

```diff
@@ -233,15 +232,15 @@ def slide(
         # prepare (and count) sliding audio chunks
         step_size: int = round(self.step * sample_rate)
         chunks: torch.Tensor = rearrange(
-            waveform.unfold(0, window_size, step_size),
-            "chunk channel frame -> chunk frame channel",
+            waveform.unfold(1, window_size, step_size),
+            "channel chunk frame -> chunk channel frame",
```

Comment:
I need to study that carefully. It already literally took me hours to understand this original `unfold` call.